client/Scenarios/ListAllResourceGroupsByName.cs

```diff
@@ -0,0 +1,24 @@
+using azure_proto_compute;
```

Comment:
Is this file needed any longer?  Since listing resource groups by name was deemed not possible in azure API?

Reply:
Nope, can remove